src/Manager/MapManager.php

```diff
@@ -128,7 +128,7 @@ public function prepareMap(int $mapId, array $config = [], Collection $overlays
         $map->setVariable('map_'.$mapId.'_'.substr(md5(time().$mapId), 0, 8));
 
         // apply map config
-        $htmlId = $mapConfig->htmlId ?: 'map_canvas_'.uniqid();
+        $htmlId = $mapConfig->htmlId ?: 'map_canvas_'.uniqid('', true);
```

Comment:
I needed to revert that because the id then contained a dot leading to css selectors not working amymore.

Reply:
okay, sorry for that! It was a security hint of the inspector